### PR TITLE
Making gem support utf8 chars

### PR DESCRIPTION
**Pull request by vlamy**
Tested in CLI, and successfully enables support for korean chars at least.

### CODE REVIEW

**Comment by rsempe** [comment]
Could you explain why you made these changes? It's difficult to review like this :|

File: lib/tm4b/broadcast.rb [hunk 1]
```diff
@@ -40,9 +40,7 @@ def split_method=(method)
       end
 
       def encoded_message
-        @message.
-          encode!("ISO-8859-1", :invalid => :replace, :undef => :replace, :replace => "").
-          encode!("UTF-8")
+        @message.encode!("UTF-8")
```
Comment (vlamy) [inline lib/tm4b/broadcast.rb]: Here, we were filtering out all non `ISO-8859-1` (latin alphabet) characters, then re-encoding the message in utf-8. Now we want to be able to send non-latin characters such as korean characters for Dior. So we do want to remove the latin alphabet filter.

**Comment by mehdi-d** [comment]
📞

**Comment by lminaudier** [comment]
:shipit:

**Pull request merged**